ui/page/staking/vsp_selector.go

```diff
@@ -44,7 +45,8 @@ func (v *vspSelector) Changed() bool {
 }
 
 func (v *vspSelector) selectVSP(vspHost string) {
-	for _, vsp := range v.WL.MultiWallet.VspList {
+	vsps := v.WL.MultiWallet.KnownVSPs()
+	for _, vsp := range vsps {
```

Comment:
```suggestion
	for _, vsp := range v.WL.MultiWallet.KnownVSPs() {
```

I dont think there is need assigning this to a variable when it is only used once.